index.js

```diff
@@ -52,7 +54,13 @@ module.exports = function parseDate (isoDate) {
 
 function getDate (isoDate) {
   var matches = DATE.exec(isoDate)
+
   var year = parseInt(matches[1], 10)
+  var isBC = !!matches[4]
+  if (isBC) {
```

Comment:
isBC is calculated elsewhere with a second regex. Fine with using the match from the main one, but please update the other fn to match and remove `BC`.